demos/icon-toggle.scss

```diff
@@ -75,6 +75,10 @@
   @include mdc-states(black);
 }
 
+#custom-on-dark {
+  background-color: #6200ee;
```

Comment:
Can we just remove this rule altogether?

The purpose of this example is to show how to customize the color of the icon; a background color isn't really necessary for that.

FYI, a red icon against a dark purple background has a [contrast ratio of `1.8`](http://leaverou.github.io/contrast-ratio/#%23de442c-on-%236200ee), which is really hard to see.
[The spec](https://material.io/guidelines/usability/accessibility.html#accessibility-color-contrast) says small text and icons should have a contrast ratio of at least `4.5:1` against their background.

Also, in Sass files, we should use the official Material color palette variables instead of hard-coding the RGB. E.g.:

```sass
#custom-on-dark {
  background-color: $material-color-grey-100;
}
```